### PR TITLE
Feature/aa 678 move email templates to edo database step2

### PR DESCRIPTION
**Pull request by AlexeyShok**
Move email templates to edo database. Step 2. Draft only, need change data in migration and remove unused mail options from startup

### CODE REVIEW

File: HappyTravel.Edo.DirectApi/Services/Dummies/EdoDummyNotificationService.cs [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 using HappyTravel.Edo.Api.Models.Users;
 using HappyTravel.Edo.Api.NotificationCenter.Models;
 using HappyTravel.Edo.Api.NotificationCenter.Services;
 using HappyTravel.Edo.Notifications.Enums;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
 
 namespace HappyTravel.Edo.DirectApi.Services.Dummies
 {
```
Comment (kirillta) [inline HappyTravel.Edo.DirectApi/Services/Dummies/EdoDummyNotificationService.cs:13]: The class name is quite non-descriptive

File: Api/Services/PropertyOwners/BookingConfirmationService.cs [hunk 2]
```diff
@@ -173,12 +173,11 @@ public async Task<Result> SendConfirmationEmail(Booking booking)
             return await _notificationService.Send(apiCaller: new Models.Users.ApiCaller(id: default, type: ApiCallerTypes.PropertyOwner),
                     messageData: bookingConfirmationData,
                     notificationType: NotificationTypes.PropertyOwnerBookingConfirmation,
-                    emails: new List<string> { emails[0], _options.ReservationsOfficeBackupEmail },
+                    emails: new List<string> { emails[0], _options.ReservationsOfficeBackupEmail });
```
Comment (mingazhev) [inline Api/Services/PropertyOwners/BookingConfirmationService.cs:176]: Why not using "First()"?

**Review by mingazhev — APPROVED** [review]
With a small comment

File: Api/Services/PropertyOwners/BookingConfirmationService.cs (hunk 2 above)
Comment (AlexeyShok) [reply Api/Services/PropertyOwners/BookingConfirmationService.cs]: Fixed

**Pull request merged**